roles/serf/files/serf.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=Serf
-After=auditd.service systemd-user-sessions.service time-sync.target
+After=network.target auditd.service systemd-user-sessions.service time-sync.target
```

Comment:
@mapuri I tried with the below, it does not seem to hit the exit at all. so, not iptables entries installed post reboot

```
    echo setting up iptables for serf
    ( /sbin/iptables -L INPUT | grep 'serf discovery sport' || \
        ( /sbin/iptables -I INPUT 1 -p udp --sport 5353 -i enp133s0 -j ACCEPT -m comment --comment 'serf discovery sport' || exit 1) )
    ( /sbin/iptables -L INPUT | grep 'serf discovery dport' || \
        ( /sbin/iptables -I INPUT 1 -p udp --dport 5353 -i enp133s0 -j ACCEPT -m comment --comment 'serf discovery dport' || exit 1) )
    ( /sbin/iptables -L INPUT | grep 'serf control' || \
        ( /sbin/iptables -I INPUT 1 -p tcp --dport 7946 -i enp133s0 -j ACCEPT -m comment --comment 'serf control' || exit 1) )
```